.ci/cico_updates_minishift.sh

```diff
@@ -134,10 +131,6 @@ testUpdates() {
   # Update the operator to the new release
   chectl server:update --skip-version-check --installer=operator --platform=minishift --che-operator-image=quay.io/eclipse/che-operator:${lastPackageVersion} --templates="tmp"
 
-# Patch images and tag the latest release
-  oc patch checluster eclipse-che --type='json' -p='[{"op": "replace", "path": "/spec/auth/identityProviderImage", "value":"quay.io/eclipse/che-keycloak:'${lastPackageVersion}'"}]' -n ${NAMESPACE}
-  oc patch checluster eclipse-che --type='json' -p='[{"op": "replace", "path": "/spec/server/devfileRegistryImage", "value":"quay.io/eclipse/che-devfile-registry:'${lastPackageVersion}'"}]' -n ${NAMESPACE}
-  oc patch checluster eclipse-che --type='json' -p='[{"op": "replace", "path": "/spec/server/pluginRegistryImage", "value":"quay.io/eclipse/che-plugin-registry:'${lastPackageVersion}'"}]' -n ${NAMESPACE}
```

Comment:
Why we remove this patch for keycloak, devfile and plugin registry?

Reply:
It makes e2e test unstable. Because, we could start workspace while pods with image patching was not redeployed yet:  che image was redeployed with desired version, but all another components are in the redeploy progress and in general Che is not ready. We could save this patching, but than we should modify "Che ready" check(https://github.com/eclipse/che-operator/blob/aeef0ef21ab482d6f35a686dc4be5d5ce15bda5d/.ci/cico_updates_minishift.sh#L86). We should wait while all che pod images are ready and has desired image version. And only after that we should try to  start test workspace.

Reply:
I replaced images defaults(in the operator.yaml) using `sed` and I applied --template argument to use current che-operator  templates in the chectl. I think It will be better for test performance and stability.

Reply:
I'm agree! Thank you for the explanation!